admin-manual/installation-setup/installation/scripts/am19-bionic-deb.sh

```diff
@@ -31,6 +31,17 @@ sudo apt-get install -y archivematica-storage-service
 sudo rm -f /etc/nginx/sites-enabled/default
 sudo ln -s /etc/nginx/sites-available/storage /etc/nginx/sites-enabled/storage
 
+#Fix for mysql UTF8 problems
+sudo bash -c 'cat << EOF >  /etc/mysql/conf.d/charset.cnf
+[mysqld]
+character-set-server = utf8
```

Comment:
Hi Santi, I think it is a good idea to add the following lines to the mysqld section:
<pre>
collation_server = utf8_general_ci
character-set-client-handshake = FALSE
</pre>
Adding `character-set-client-handshake = FALSE` ignores the character settings from client, so you can remove the below client line.